pipeline/extract_carbon.py

```diff
@@ -61,24 +65,24 @@ class CustomDataProcessor(DataProcessor):
     """
     Custom DataProcessor that implements a specific process_data method.
     """
-    
-    def __init__(self, save_location: str = SAVE_LOCATION, 
-                 aws_access_key: str = AWS_ACCESS_KEY, 
-                 aws_secret_key: str = AWS_SECRET_KEY, 
-                 region: str = AWS_REGION, 
-                 s3_file_name: str = SAVE_NAME, 
-                 bucket: str = S3_BUCKET, 
+
+    def __init__(self, save_location: str = SAVE_LOCATION,
```

Comment:
I assume this was auto linted, good to remove whitespace. Not sure where it comes from.